pkg/operator/starter.go

```diff
@@ -68,6 +72,24 @@ func RunOperator(ctx context.Context, kubeconfig *restclient.Config) error {
 	}
 	eventRecorder := events.NewKubeRecorder(kubeClient.CoreV1().Events(defaults.ImageRegistryOperatorNamespace), "image-registry-operator", controllerRef)
 
+	desiredVersion := status.VersionForOperatorFromEnv()
+	missingVersion := "0.0.1-snapshot"
+
+	// By default, this will exit(0) the process if the featuregates ever change to a different set of values.
+	featureGateAccessor := featuregates.NewFeatureGateAccess(desiredVersion, missingVersion, configInformers.Config().V1().ClusterVersions(), configInformers.Config().V1().FeatureGates(), eventRecorder)
+
+	go featureGateAccessor.Run(ctx)
```

Comment:
should we give `ctx.Done` to `Run` like the other calls to `Run` do down below? i.e:
```
go featureGateAccessor.Run(ctx.Done())
```

Reply:
Never mind - `ctx.Done()` is a channel, and `fetureGateAccessor.Run` takes a regular context parameter.